### PR TITLE
react-react2-week1 HW github user first commit

### PR DESCRIPTION
**Pull request by agkabir**

### CODE REVIEW

File: react/react2/react2-week1-githubuser/src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const Header = () => {
+  return <h1> Github user searcher</h1>;
```
Comment (chezzoba) [inline react/react2/react2-week1-githubuser/src/components/Header.jsx:2]: Why do this when you can just put in the h1 in the App function directly?

File: react/react2/react2-week1-githubuser/src/components/UserContext.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  useEffect(() => {
+    if (searchText) {
+      const fetchUsers = async () => {
+        try {
+          setIsLoading(true);
+          const response = await fetch(api_url + searchText);
+          if (!response.ok) throw Error("Did not receive expected data!");
+          const result = await response.json();
+          setUserList(result.items);
+          setFetchError(null);
```
Comment (chezzoba) [inline react/react2/react2-week1-githubuser/src/components/UserContext.jsx]: Really good error handling logic here.

**Review by chezzoba — APPROVED** [review]
I am very impressed by your coding ability Kabir.

File: react/react2/react2-week1-githubuser/src/components/Header.jsx (hunk 1 above)
Comment (agkabir) [reply react/react2/react2-week1-githubuser/src/components/Header.jsx:2]: Thats true.

**Comment by agkabir** [comment]
By the way thank you for the nice comment.🙂

**Pull request merged**